appscale/tools/appscale.py

```diff
@@ -177,7 +177,10 @@ def get_nodes(self, keyname):
     """
     try:
       with open(self.get_locations_json_file(keyname)) as locations_file:
-        return json.loads(locations_file.read())
+        json_local_nodes = json.loads(locations_file.read()).get('node_info')
```

Comment:
`get` has a `default` parameter. To simplify, you can just `return json.loads...get('node_info', default=[])`